redisinsight/ui/src/pages/vector-search/create-index/hooks/useCreateIndex.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { renderHook, act } from '@testing-library/react-hooks'
+import { useCreateIndex } from './useCreateIndex'
+import { SampleDataType, SearchIndexType } from '../types'
+
+const mockLoad = jest.fn()
+const mockDispatch = jest.fn()
+
+jest.mock('uiSrc/services/hooks', () => ({
+  useLoadData: () => ({
+    load: mockLoad,
+  }),
+  useDispatchWbQuery: () => mockDispatch,
+}))
+
+jest.mock('uiSrc/utils/index/generateFtCreateCommand', () => ({
+  generateFtCreateCommand: () => 'FT.CREATE idx:bikes_vss ...',
+}))
+
+describe('useCreateIndex', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  const defaultParams = {
```

Comment:
We can type-hint it :) 

```suggestion
  const defaultParams: CreateSearchIndexParameters = {
```